docs/README.md

```diff
@@ -18,10 +18,51 @@ Navigate your shell to your project's directory. You should already have install
 
 ### Experimental Features
 
+#### Mutagen
+
 You may find that file sharing performance or server response times are slower than you would like on Windows or MacOS. Local Server provides an experimental integration with [Mutagen](https://mutagen.io/) to resolve this.
 
 See the [Mutagen set up guide for detailed instructions on how to install and run it](./mutagen-file-sharing.md).
 
+#### Subdomain and Custom domains in multisites
```

Comment:
Do we want this under experimental features? Custom domains / codespaces perhaps but I think the SSL stuff itself doesn't need to be considered experimental as we need it to work for the default TLD at the very least.

Reply:
It's experimental in the sense that it is tied to custom domains which is experimental 🤔

Reply:
See https://github.com/humanmade/altis-local-server/issues/475

Reply:
I think it’s fine to not list it as experimental personally, codespaces definitely will be to start with

Reply:
@roborourke Updated docs in https://github.com/humanmade/altis-local-server/pull/465/commits/2f11c98a46527f83c2045563133c2ad948c0a547 , should be ready for a final review.